tools/cldr-apps/src/main/java/org/unicode/cldr/web/MailSender.java

```diff
@@ -72,6 +72,9 @@
                 SurveyMain.isUnofficial() ? 5 : 60);
             // Check for outbound mail every 60 seconds
             CLDR_MAIL_DELAY_EACH = CLDRConfig.getInstance().getProperty("CLDR_MAIL_DELAY_EACH", 60);
+
+
+            System.err.println("Logger settings " + logger.getLevel().toString() + " for " + logger.getName());
         }
```

Comment:
I'm assuming this println won't happen often enough to clog the logs, and/or it's temporary -- also ideally shouldn't go to ".err" if it's not an error

Reply:
Temporary. I'll delete it

Reply:
@btangmu actually that is printed exactly one time at startup. Will remove in the PR.